lib/membrane_rtmp_plugin/rtmp/sink/sink.ex

```diff
@@ -253,7 +253,11 @@ defmodule Membrane.RTMP.Sink do
     end
   end
 
-  defp write_frame_interleaved(%{frame_buffer: %{audio: audio, video: video}} = state)
+  defp write_frame_interleaved(
+         %{
+           frame_buffer: %{{Membrane.Pad, :audio, 0} => audio, {Membrane.Pad, :video, 0} => video}
```

Comment:
```suggestion
           frame_buffer: %{Pad.ref(:audio, 0) => audio, Pad.ref(:video, 0) => video}
```

Reply:
Thanks for help me with this. and sorry for take your time to guide me.

Reply:
No need to be sorry, you are more than fine :)